tests/integrate/190_PW_KPAR/jd

```diff
@@ -0,0 +1 @@
+test parallel method kpar
```

Comment:
This case must runned by macro __MPI, and number of cores used for this case must larger than 2, please add this warning to file jd.

Reply:
ok